### DISCUSSION TITLE
Is this still under active development?

### DISCUSSION THREAD

**Discussion by emigre459** [stellargraph/stellargraph#1919] [General]
Hi, I'm excited to use StellarGraph, but I have a question: is this package still under active development? I mainly ask because it looks like there was a very steady release cadence through June 2020, but then no releases since then.  It looks like some branches are being actively worked on, but the main branch is pretty inactive. 

If there is another package that is now the successor to StellarGraph or something, that would be great to know about. It's an awesome resource to be sure!

**Comment by CalmLogarithm** [comment]
A valid question. More recently the interactions here under Discussions have trailed off as well.
